setup.md

```diff
@@ -0,0 +1,322 @@
+
+In order to ensure that you have everything ready in advance of the workshop,
+we ask that you follow these setup instructions before attending each tutorial.
+Q&A sessions will be setup on the two days preceeding the course to help you
+with any issues you might encounter during the setup process.
+
+
+## 1. [Anaconda]
+
+All tutorials in this course rely heavily on the [Anaconda] package manager.
+This cross platform tool allows you to install other software packages in an
+easy and controllable manner. We require all attendees to install this prior
+to starting any of the tutorials. Please see the follow the [Anaconda install instructions]
+to get started.
+
+The [Anaconda] package manager, is named [conda]. Throughout the tutorials you
+will see several invocations of [conda], including to create [environments] and
+install software. If you are unfamiliar with this tool, it is heavily
+recommened that you go through the [getting started guide].
+
+
+## 2. Using the terminal
+
+Throughout the tutorials we will be interacting with various software through
+the use of a terminal interface. Terminals are a powerful means by which to
+navigate through files and quickly execute various programs.
+
+On linux/macOS this is generally done though a bash/zsh shell.
+
+- In linux this can be accessed by searching for terminal in the startup menu.
+  See below for an example in the ubuntu gnome shell.
+
+![Ubuntu shell](./images/main-setup/ubuntu-terminal.png)
+
+- On macOS this is done by searching for terminal in the Launchpad (see the
+  [following for more details][apple terminal support]).
+
+(Todo: Insert picture of opening terminal in macOS here)
+
+On Windows (unless using WSL in which case please follow the linux instructions),
+this will be done via the Anaconda prompt. The Anaconda prompt can be opened by
+searching for it in the startup menu after having installed Anaconda (see the
+step above).
+
+![Anaconda Prompt](./images/main-setup/anaconda-terminal.png)
+
+There are substantial differences in the way in which the terminal works between
+Windows and linx/macOS. We will give a quick overview of the the basics below,
+including some links for extra resources you can look at to get more familiar
+with using the shell. We recommend that you have a look through these before
+attempting the tutorials.
+
+Note: in the MD tutorial you will eventually be required to access a remote
+HPC machine to run a Molecular Dynamics simulations. This will require use
+of the linux terminal. If you are using Windows you will therefore also need
+to look at the instructions for the linux terminal before attempting this tutorial.
+
+### Basic linux/macOS terminal instructions
+
+This section only provides a very basic overview of using the bash/zsh shell
+in linux/macOS. For more information please see the following tutorials:
+
+- [Ubuntu's terminal tutorial](https://ubuntu.com/tutorials/command-line-for-beginners#1-overview)
```

Comment:
Anyone got a MacOS equivalent tutorial?

Reply:
I would just point them at one of the youtube vids - this one is OK

https://www.youtube.com/watch?v=aKRYQsKR46I

Reply:
Thanks I'll do something similar for the windows cmd terminal too.